components/admin_console/license_settings/enterprise_edition/enterprise_edition_left_panel.tsx

```diff
@@ -117,10 +133,16 @@ const EnterpriseEditionLeftPanel = ({
             <div className='licenseInformation'>
                 <div className='license-details-top'>
                     <span className='title'>{'License details'}</span>
-                    {(expirationDays <= 30) &&
-                        <span className='expiration-days'>
-                            {`Expires in ${expirationDays} day${expirationDays > 1 ? 's' : ''}`}
-                        </span>
+                    {(canExpand && expirationDays > 60) &&
```

Comment:
Definitely will make it easier to follow on the front end. I don't see any downsides to removing this and just using the self-serve status to determine whether or not we display this, I'll make the necessary changes to this pr.

> there's nothing technically wrong with a backend request to expand within 60 days of expiration

Definitely nothing wrong with this, and yes this was discussed in sprint panning.

Reply:
There is currently an endpoint that checks if you are able to expand in the portal https://github.com/mattermost/customer-web-server/blob/31ceeaaba3907a3b2fb39f6256ab594e0100296e/api/license.go#L41

Even though I believe they should renew instead of expand if they are in the renewal window, @johndavidlugtu was going to provide the final direction on how restrictive we should be with this condition

Reply:
Hey team -- OK with doing renewal only instead of expansion during the renewal window knowing that the renewal license will contain 12 months + remaining time of existing license

cc: @marianunez @neallred @ConorMacpherson